Marketplace/Cardz/Card.js

```diff
@@ -141,101 +109,66 @@
         var checkIfCard = properties.card;
         var held = properties.held;
         var deckHandlerID = properties.deckHandlerID;
-        if ((checkIfCard == true) && (held == false) && (hand == "right")) {
-            //feedback to let you know you attached the card to your hand
-            Controller.triggerShortHapticPulse(.9, 1);
-            //get position of left and right hand
-            var left = MyAvatar.getLeftPalmPosition();
-            var right = props.position;
-            //get distance between right hand and left hand
-            var dx = left.x - right.x;
-            var dy = left.y - right.y;
-            var dz = left.z - right.z;
-            var dist = getDistance(dx, dy, dz);
-            //if card is within an acceptable distance to your hand then place it
-            if (dist < .10) {
-                //place card and change held state
-                var placement = {
-                    userData: JSON.stringify({
-                        grabbableKey: {
-                            grabbable: true,
-                            ignoreIK: false
-                        },
-                        "held": true,
-                        "card": true,
-                        "deckHandlerID": deckHandlerID
-                    }),
-                    parentID: MyAvatar.sessionUUID,
-                    parentJointIndex: leftHandJoint
-                };
-                Entities.editEntity(_this.entityID, placement);
-            //reset card if not close enough
-            } else if (dist >= .10) {
-                //unparent to hand and change held state
-                var unparentToMe = {
-                    parentID: ""
-                };
-                Entities.editEntity(_this.entityID, unparentToMe);
-                //send message
-                var cardChannel = "card-channel-".concat(deckHandlerID);
-                var data = [true, _this.entityID, MyAvatar.sessionUUID];
-                Messages.sendMessage(cardChannel, JSON.stringify(data));
-                //delete an overlay if there is one
-                try {
-                    Overlays.deleteOverlay(cardOverlay);
-                    cardOverlay = undefined;
-                } catch (err) {
-                    //e
+        //check to see if you are holding a card that is not already being held

```

Comment:
This also seems to be repeated code between left and right hand that can be reduced